package.json

```diff
@@ -1,31 +1,47 @@
 {
-  "name": "Travel_journal",
-  "version": "1.0.0",
-  "description": "",
+  "name": "webpack-boilerplate",
+  "version": "2.0.0",
+  "description": "Sensible webpack 5 boilerplate using Babel and PostCSS with a hot dev server and an optimized production build.",
   "main": "index.js",
+  "author": "Tania Rascia",
+  "license": "MIT",
   "scripts": {
-    "dev": "parcel src/index.html",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "cross-env NODE_ENV=development webpack serve --config config/webpack.dev.js",
+    "build": "cross-env NODE_ENV=production webpack --config config/webpack.prod.js"
```

Comment:
Hey @majac91, have you tried this project in production mode (`npm run build`)? It seems to have the same problem as with Parcel. So it seems the problem is within Parse itself.

```
Uncaught TypeError: Incompatible receiver, Map required
```
![image](https://user-images.githubusercontent.com/14869087/101988756-ab7eb900-3c93-11eb-92bf-cb07b1fcb53d.png)


I'd recommend you to open an issue to [Parse JS](https://github.com/parse-community/Parse-SDK-JS) with this problem. Explain the problem following their instructions and leave a link to this repo for them to try it out. They might be able to answer and solve the error.

Reply:
Don't worry, I don't get demotivated that easily 😄 I'll open an issue and ask but I'd rather start with Firebase than wait too long for answers

Reply:
Okay. So, create the issue poiting to this branch (even this comment if you want to). 

And meanwhile, open a new PR and start working on Firebase ;)

Reply:
Hey, I just checked the Travel journal project and the latest deploy (Webpack-migration @b4564da) works just fine,  no Parse issues 🤔

Reply:
However trying to deploy this project from main I ran into an error 
<img width="1189" alt="Screenshot 2020-12-13 at 14 08 03" src="https://user-images.githubusercontent.com/62510606/102012797-acbfec80-3d4c-11eb-8763-7d7cee615017.png">
The config and build settings are the same and I double checked the file name.

Reply:
Wow, it really worked on the travel journal... Have you changed anything there? Which commit exactly did you publish?

Regarding the error, it explicit says `"no such file or direction (...) src/img/photo0.jpg'`. So, it's importing a file that does not exist.

Reply:
That's the point, the file does exist and the path looks okay to me. Is it wrong?  This is the published travel journal commit @b4564da.

Reply:
Hi @majac91,

The screenshot you shared with the error uses `parcel`. Are you in the correct branch (this one with webpack?)

I ran the project locally, this branch, latest pushed changes, and the `build` script seemed to work fine, without issues. ![See screenshot](https://user-images.githubusercontent.com/14869087/102992514-40df4000-4513-11eb-8e5a-be83c85cdb6d.png).